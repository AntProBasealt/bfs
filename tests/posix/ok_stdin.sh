# -ok should *not* close stdin
# See https://savannah.gnu.org/bugs/?24561
yes | bfs_diff basic -ok bash -c 'printf "%s? " "$1" && head -n1' bash {} \;
