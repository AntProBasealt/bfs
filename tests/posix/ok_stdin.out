basic/a? y
basic/b? y
basic/c/d? y
basic/c? y
basic/e/f? y
basic/e? y
basic/g/h? y
basic/g? y
basic/i? y
basic/j/foo? y
basic/j? y
basic/k/foo/bar? y
basic/k/foo? y
basic/k? y
basic/l/foo/bar/baz? y
basic/l/foo/bar? y
basic/l/foo? y
basic/l? y
basic? y
